data/divers/divers . hotellerie.yaml

```
@@ -0,0 +1,459 @@
+  
+divers . hotellerie:
+  formule: caractéristique * nuits
+  icônes: 🛌
+  unité: kgCO2e
+  note: |
+    Nous nous interessons ici uniquement aux émissions liées à la consommation de services d'hôtellerie (hôtels et hébergements similaires). Les airbnb ou les 
+    logements plus sommaires comme les campings et bungalows ne sont encore pas pris en compte
+  
+divers . hotellerie . nuits:
+  question: Sur une année, combien de nuits passez-vous à l'hotêl ou dans des hébergements du même type autre que votre logement (vacances incluses) ?
```

```suggestion
  question: Sur une année, combien de nuits passez-vous dans un hôtel (vacances incluses) ?
```
Je ne vois pas l'utilité de `ou dans un hébergement du même type`, sachant qu'on a précisé avant que les AirBnB ou camping n'était pas pris en compte. De quel `même type` parlons nous ?  (+ typo sur hôtel)